### DISCUSSION TITLE
Allow user creation by admin

### DISCUSSION THREAD

**Discussion by mkevenaar** [OctoFarm/OctoFarm#528] [Implemented]
Hi,
As a sysadmin I would like to be in control on the account creation. 
If you disable the registration, there is currently no way to create an account for another user.
I would like to suggest to implement an form in the Admin section to create an new user.

**Comment by NotExpectedYet** [comment]
> Hi, As a sysadmin I would like to be in control on the account creation. If you disable the registration, there is currently no way to create an account for another user. I would like to suggest to implement an form in the Admin section to create an new user.

This is coming in V1.2 :)
